app/templates/gulpfile.js

```diff
@@ -473,17 +482,27 @@ function serve(isDev, specRunner) {
     });
 }
 
-function getNodeOptions(isDev) {
-  var port = args['app-port'] || process.env.PORT || config.defaultPort || 9070;
+/**
+ * Determine whether the enviornment requires dev mode'
+ * @param {String} env - 'local', 'dev', or 'prod'
+ * @return {Boolean} - true if env==='local'
+ */
+function isDevMode(env) {
+  return env === 'local';
+}
+
+function getNodeOptions(env) {
+  var envJson = require('./deploy/gulp-' + env + '.json');
```

Comment:
Won't the require break if gulp-local.json doesn't exist? If yes, perhaps try to ignore the error, and maybe print a message suggesting you can provide local overrides with that file.